src/aioswitcher/api/packets.py

```diff
@@ -17,16 +17,29 @@
 # weekdays sum, start-time timestamp, end-time timestamp
 SCHEDULE_CREATE_DATA_FORMAT = "01{}01{}{}"
 
+PHONE_ID = "0000"
+DEVICE_PASSWORD = "00000000"
```

Comment:
bandit is complaining about hardcoded password and we don't use the `PHONE_ID` and `DEVICE_PASSWORD` anyhow so we can do the same as we do for other devices by replacing all zeros with padding:
We can update also the older ones to use 72 padded zeros and all packets will have the same paddings:

`SEND_CONTROL_PACKET = "fef05d0002320102" + REQUEST_FORMAT + "{}" + PAD_72_ZEROS + "000106000{}00{}"`